src/tests/.gitignore

```diff
@@ -0,0 +1,3 @@
+.*.*
+.*.*
```

Comment:
can't believe I'm actually commenting on this PR, but these 2 lines are identical